ui/appui-react/src/appui-react/icons/cube-faces/SvgCubeFaceBottom.tsx

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Utilities
+ */
+
+import * as React from "react";
+
+/** @internal */
+export function SvgCubeFaceBottom() {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+      <path
+        opacity=".33"
+        style={{ isolation: "isolate" }}
```

Comment:
I see this `style={{ isolation: "isolate" }}` in a couple of places.  Did you add this yourself?  If you did not add this, you may be able to remove this as well.  [This CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/isolation) is typically used in combination with `z-index`, not sure if this is needed.

Reply:
This property was in SVGs before `style="isolation:isolate"`, removed them now.